apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/HealthResource.java

```diff
@@ -22,13 +22,20 @@
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 
+/** Legacy Health endpoint used by some routers to check if the service is up and running. */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
 @Singleton
 public class HealthResource {
   @GET
-  @Path("/health")
+  @Path("health")
```

Comment:
JAX-RS does not differentiate between leading slash and missing, other frameworks (Vertex) may.
Seems more correct not to indicate absolute path but related to class annotation.